roles/zabbix_web/defaults/main.yml

```diff
@@ -105,6 +105,8 @@ zabbix_server_dbname: zabbix-server
 zabbix_server_dbuser: zabbix-server
 zabbix_server_dbpassword: zabbix-server
 zabbix_server_dbport: 5432
+zabbix_server_dbencryption: "false"
```

Comment:
@dj-wasabi I think that this bool is not related to the Ansible itself nad should be propagated to the zabbix.conf.php as a "false":

https://git.zabbix.com/projects/ZBX/repos/zabbix/browse/ui/conf/zabbix.conf.php.example

So this patch can probably be merged it it is still needed. I will leave that up to you

Reply:
Yes, but I don't like to use booleans as string, what if someone is actually configuring it as a boolean. I would then prefer to use it as a boolean and write that in the template as a string.

So:
```
zabbix_server_dbencryption: false
```
Which results in:
```
$DB['VERIFY_HOST'] = 'false';
```
I think it would be as simple as:
```
$DB['VERIFY_HOST'] = '{{ zabbix_server_dbverifyhost | string | lower }}';
```
Even if someone provides a `False` or a `True`.

Reply:
That makes sense :)

@YoSiJo will you please have a chance to change it like @dj-wasabi defined?